### PR TITLE
Support sending Websocket Ping and ignoring Pong responses

### PR DESCRIPTION
**Pull request by nizar-m**

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -1246,6 +1257,7 @@ Fine tuning of the installation directories:
   --sysconfdir=DIR        read-only single-machine data [PREFIX/etc]
   --sharedstatedir=DIR    modifiable architecture-independent data [PREFIX/com]
   --localstatedir=DIR     modifiable single-machine data [PREFIX/var]
+  --runstatedir=DIR       modifiable per-process data [LOCALSTATEDIR/run]
```
Comment (tisba) [inline configure]: I don't follow why the changes to `configure` are required for this PR. Looks like they were added by accident?

**Comment by nizar-m** [comment]
A version which sends a ping and expects a pong back as a response, should be useful.

This is not doing that.

**Comment by tisba** [comment]
Okay, understood. Thanks for your half-way-hacky thing anyway as this might be useful if someone what's to work on this feature 👍
